### ISSUE TITLE
Web.py - google maps

### ISSUE THREAD

**Issue by Baklap4** [modrzew/pokeminer#221]
Instead of using open street maps wouldny it be a good thing to use google maps with the api key suplied in the config?
At the moment we're using google maps in the report view and open streets map in the map view.
Just use one service and personally i favor google maps over open street maps.

**Comment by Aiyubi** [comment]
You are not here for long :D 
read this changelog second point https://github.com/modrzew/pokeminer/releases/tag/v0.5.0

**Comment by Baklap4** [comment]
I am indeed not.
But then if open streets map is a very nice tool, why not create a heat map just with open streets maps too?
I find it kind of ugly to use two different map providera when both can do the same things desired. As of the release notes id suggest to change completely to osm and change the heatmaps to use osm as well :) and by this getting rid od the google api key

**Comment by YonderGod** [comment]
It's a work in progress.

**Comment by paul30001** [comment]
@Baklap4 I agree with your 2nd Comment, especially since my API makes Google Maps be in Russian for some reason, and I can't change it.

**Comment by modrzew** [comment]
@Baklap4 having a Google Maps API key in the config is *precisely* the reason I switched to Leaflet. Simply because their daily request quota is very low and you run out fast. Also IMO Leaflet has easier JS API and is faster than Google Maps.

Plugging reports pages into Leaflet is on my continuously-growing todo list - then you shouldn't need any API key anymore.
